src/pages/NewGroupPage.js

```diff
@@ -248,16 +250,25 @@ class NewGroupPage extends Component {
                         }}
                         headerTitle={headerTitle}
                         headerMessage={headerMessage}
+                        isOnRightDockedModal
                     />
-                    <TouchableOpacity
-                        onPress={this.createGroup}
-                        style={[styles.button, styles.buttonSuccess, styles.w100, styles.mt5]}
-                    >
-                        <Text style={[styles.buttonText, styles.buttonSuccessText]}>
-                            Create Group
-                        </Text>
-                    </TouchableOpacity>
+                    <View style={[styles.ph5, styles.pb5]}>
+                        <Pressable
+                            onPress={this.createGroup}
+                            style={({hovered}) => [
+                                styles.button,
+                                styles.buttonSuccess,
+                                styles.w100,
+                                hovered && {backgroundColor: themeColors.buttonSuccessHoveredBG},
```

Comment:
`{backgroundColor: themeColors.buttonSuccessHoveredBG}` should be a style if possible. 
Since we are trying to avoid having inline styles everywhere for consistency.

Reply:
Done!